src/core/viz/expressions/belongs.js

```diff
@@ -59,10 +69,14 @@ function NIN_INLINE_MAKER(categories) {
 
 function generateBelongsExpression(name, inlineMaker, jsEval) {
     return class BelongExpression extends BaseExpression {
-        constructor(value, ...categories) {
+        constructor(value, categories) {
             value = implicitCast(value);
             checkExpression(name, 'value', 0, value);
 
+            categories = categories || [];
+            if (!Array.isArray(categories)) {
+                categories = [categories];
+            }
```

Comment:
So... we admit `s.in(s.prop('type'), 'metropolis')` for `s.in(s.prop('type'), ['metropolis'])`, right? Which is nice for the user fingers... but will be allow it also in the String language? ...in all functions that take arrays? ...is it worth the effort?

Reply:
Good point! I added this code because of the tests, but the point is that we must use always [] for all these cases, so we will use arrays also in the String API.